### PR TITLE
Clarify return value

### PR DESCRIPTION
**Pull request by Flamenco**
### Description

Clarify return value when input is null or undefined.

### Motivation

Change was made because it is missing from description

### CODE REVIEW

File: files/en-us/web/javascript/reference/global_objects/regexp/test/index.md [hunk 1]
```diff
@@ -41,7 +41,8 @@ test(str)
 ### Returns
 
 `true` if there is a match between the regular expression and the string
-`str`. Otherwise, `false`.
+`str`. `false` if the `str` parameter is `null` or `undefined`. Otherwise, `false`.
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/global_objects/regexp/test/index.md:44]: There's nothing special about `null`/`undefined`; they just get coerced to strings and become `"null"`/`"undefined"`.

```js
/null/.test(null); // true
/undefined/.test(undefined); // true
```

**Comment by Josh-Cena** [comment]
Thanks @Flamenco Unfortunately the change is incorrect. In MDN's parameter description, if we tell you a function accepts a string, it usually means non-string arguments get directly coerced to strings. It's exactly the case here; there's no special processing for `null`/`undefined`.

**Comment by Flamenco** [comment]
Oh yes, I see that now.  `/undefined/` will test `true` for `test(undefined)` and `/null/` will test `true` for `/null/`.  Perhaps even that is worth mentioning however.

Thanks!

**Comment by Josh-Cena** [comment]
This is the same for almost all string-handling functions. `"".concat(null, undefined)` returns `"nullundefined"`. It's just a language feature to understand, not an API feature.

**Pull request closed without merging**